concourse/pipelines/create.yml

```diff
@@ -1184,7 +1204,8 @@ jobs:
               - -c
               - |
                 export CONCOURSE_URL="https://${CONCOURSE_HOSTNAME}.${SYSTEM_DNS_ZONE_NAME}"
-                export CONCOURSE_ATC_PASSWORD=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.concourse_atc_password concourse-secrets/concourse-secrets.yml)
+                export CONCOURSE_ATC_PASSWORD
+                CONCOURSE_ATC_PASSWORD=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.concourse_atc_password concourse-secrets/concourse-secrets.yml)
```

Comment:
It might have been always failing shellcheck. I'm not sure that we've made the warnings generated by pipecleaner fail the build (we're not passing the `--fatal-warnings` flag to pipecleaner). We probably should now that the last one has been fixed.